### PR TITLE
Don’t count SSN breaches for non-US users

### PR DESCRIPTION
**Pull request by flozia**
# References: 
Jira: [MNTOR-2871](https://mozilla-hub.atlassian.net/browse/MNTOR-2871)

# Description

Filter out SSN breach data classes for non-US users entirely.

[MNTOR-2871]: https://mozilla-hub.atlassian.net/browse/MNTOR-2871?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/functions/server/breachResolution.ts [hunk 1]
```diff
@@ -228,24 +228,4 @@ function getRecommendationFromResolution(resolution: any, args: any) {
...
-  // return type than `any` yet:
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-  return originalDataTypes.filter((d) => relevantDataTypes.includes(d));
-}
-
-export {
-  BreachDataTypes,
-  appendBreachResolutionChecklist,
-  filterBreachDataTypes,
-};
```
Comment (flozia) [inline src/app/functions/server/breachResolution.ts:251]: This function is not being used anywhere.

Comment (Vinnl) [reply src/app/functions/server/breachResolution.ts:251]: giphy

**Review by Vinnl — APPROVED** [review]

**Comment by Vinnl** [comment]
(Both the ticket number linked in the description, as well as the one in the branch name, appear to be the wrong ticket though :sweat_smile: )

**Comment by flozia** [comment]
> (Both the ticket number linked in the description, as well as the one in the branch name, appear to be the wrong ticket though 😅 )

Dang! Thanks for the catch — updated the ticket number to [MNTOR-2871](https://mozilla-hub.atlassian.net/browse/MNTOR-2871).

**Pull request merged**